cli/cli.ts

```diff
@@ -6966,6 +6967,11 @@ ${pxt.crowdin.KEY_VARIABLE} - crowdin key
             noauth: {
                 description: "disable localtoken-based authentication",
                 aliases: ["na"],
+            },
+            backport: {
```

Comment:
Should we include an alias off the bat, like `bp`?

Reply:
It handles an uncommon dev scenario, so probably not worth allocating an alias.